src/components/StartBillContainer/StartBillContainer.jsx

```diff
@@ -0,0 +1,38 @@
+import React, {Component} from "react";
+import "./styles.scss";
+import StartBillButton from "../StartBillButton/StartBillButton";
+
+class StartBillContainer extends Component {
+    constructor(props) {
+        super(props);
+        this.handleStartBill = this.handleStartBill.bind(this);
+    }
+
+    handleStartBill(billId) {
+        this.props.startABill(billId);
+    }
+
+    billCanBeStarted() {
+        return this.props.userEmail === this.props.bill.responsible.email
+            && this.props.bill.status === "OPEN";
+    }
+
+    render() {
+        const {bill} = this.props;
+
+        const StartActiveBill = billId => {
+            return (
+                <div className="button__container">
+                    <StartBillButton onClickHandler={() => this.handleStartBill(billId)}/>
+                </div>
+            )
+        }
+
+        const displayStartBillButton = this.billCanBeStarted() ? (StartActiveBill(bill.id)) : (<div/>);
```

Comment:
no point in having `<div/>`.

`this.billCanBeStarted() && (StartActiveBill(bill.id))`

`StartActiveBill(bill.id)` if true, else null

Reply:
not sure about that because we're using `justify-content: space-between` so if there's not another tag, it won't push it to the left